e2e-tests/IIoTPlatform-E2E-Tests/Standalone/C_PublishMultipleEndpointsStandaloneDirectMethodTestTheory.cs

```diff
@@ -474,6 +535,18 @@ IIoTMultipleNodesTestContext context
 
             Assert.Equal((int)HttpStatusCode.OK, response.Status);
 
+            //Call GetDiagnosticInfo direct method
+            responseGetDiagnosticInfo = await CallMethodAsync(
+                new MethodParameterModel {
+                    Name = TestConstants.DirectMethodLegacyNames.GetDiagnosticInfo,
+                },
+                cts.Token
+            ).ConfigureAwait(false);
+
+            Assert.Equal((int)HttpStatusCode.OK, responseGetDiagnosticInfo.Status);
+            diagInfo = _serializer.Deserialize<DiagnosticInfoLegacyModel>(responseGetDiagnosticInfo.JsonPayload);
+            Assert.Equal(diagInfo.NumberOfOpcSessionsConfigured, 1);
```

Comment:
Let's add a few more field checks for legacy mode as well. Just a few that we should be able to check for:

NumberOfOpcSessionsConnected == 1
NumberOfOpcSubscriptionsConfigured == 1
NumberOfOpcSubscriptionsConnected == 1
NumberOfOpcMonitoredItemsConfigured == (number of nodes that we configured)
NumberOfOpcMonitoredItemsMonitored == (number of nodes that we configured)
SentMessages > 0
FailedMessages == 0
TooLargeCount == 0

Reply:
added